### ISSUE TITLE
Not clear if this works with lodash v4 at all

### ISSUE THREAD

**Issue by butsjoh** [lodash/babel-plugin-lodash#23]
I installed the plugin and configured it in webpack but it is install lodash 3.10.1 as a dependency. If i then manually install lodash (npm install lodash --save). And then according to your plugin use something like:

```
import {isEmpty} from 'lodash';
```

i get this:

Module build failed: Error: 
/Users/butsjoh/Younited/trusted_platform/app/assets/webpack/javascripts/components/notifications.connected.jsx: lodash method isEmpty was not in known modules

**Comment by butsjoh** [comment]
Also you are claiming in the README that it is possible to include seperate functions (https://github.com/lodash/babel-plugin-lodash/blob/master/test/fixtures/multi-specifier/actual.js). I can't get this to work, could you please explain on how to setup it up properly with webpack?

what i did:

```
npm install babel-plugin-lodash-modularize --save-dev
npm install lodash --save # installed v4
```

then in my webpack config:
```
...

  module: {
    loaders: [
      {
        test: /\.jsx?$/,
        exclude: /(node_modules|bower_components|vendor)/,
        loader: 'babel',
        query: {
          plugins: ['transform-runtime', 'lodash'],
          presets: ['es2015', 'react', 'stage-0']
        }
      },
    ]
  },
```

Then in a jsx file:

```
import {isEmpty} from 'lodash';
```

Raise error: Module build failed: Error: 
/Users/butsjoh/Younited/trusted_platform/app/assets/webpack/javascripts/components/notifications.connected.jsx: lodash method isEmpty was not in known modules

**Comment by corbanbrook** [comment]
Was getting the same issue until I realized I needed to use lodash-es instead of lodash.

**Comment by megawac** [comment]
This plugin hasn't been updated for lodash v4 yet.

**Comment by megawac** [comment]
Closing as this is a dupe of #22

**Comment by megawac** [comment]
Fixed in [v2.0.1](https://github.com/lodash/babel-plugin-lodash/tree/v2.0.1)
